### ISSUE TITLE
Features to Port from Outdated Branch

### ISSUE THREAD

**Issue by cosmiccoincidence** [RE-SS3D/SS3D#1181]
## Summary

A list of systems that need to be ported from the "outdated" branch to the new one.

Each one of these will be made into a new issue when the time is right, but for now this list exists so we don't forget about any of these.

(I or other maintainers will update this issue with time)

Please ask on discord if you wish to try and import one of those, as someone might already be working on it.

To launch the outdated branch game, do the following after pressing play : 
Host -> admin -> load map (ministation is fine) -> start round and then you can embark

## Systems

The following systems are listed *roughly* based on priority.

(A couple of these may need to be rebuilt from scratch)

- **FOV** - #1102 

- **Chat** - #1286 

- **Examine** - #1257 

- **Lighting** - #1343 

- **Inventory UI** - #1346 

- **Functional Doors**

  - Airlock adjacency integration - #1106 
  - Basic open/close with lights - #1316

- **Item hold positions** - #1348 

- **Several basic interactions**

  - service bell
  - lighter
  - cigs
  - powercells
  - disposals

- **Tilemap features:**

  - Toggle visibility for individual layers - #1338 
  - Multiple tilemaps in a single map - #1341 

- **Tilemap editor features:**

  - Faster tilemap construction functions - #1056 

- **Material/tool construction** - #1450 

- **Ragdolls** - #1272 (this wasn't a port but resolved the task anyway)

- **Atmospherics** - #1464
(with basic machinery)

- **Audio** - #1320 

- **Basic consumption**

- **Floor cutout shader** - #1336 

- **Liquid shader** - #1337

- **Crayon drawing**

When porting these to the new branch, be sure to credit the original contributor (easiest way to do so is by simply linking to the reference PR/commit).

## Goal

This issue can be closed once all these tasks have their own issue made.

**Comment by stilnat** [comment]
It's a good idea to list what we need to import back, but I'd rather have it ASAP as singular issues, as we need to take a look at what need to be changed or not on each of these. Also, some are already being worked on I believe like the FOV system. I'm updating your issue to reflect that.

**Comment by cosmiccoincidence** [comment]
> It's a good idea to list what we need to import back, but I'd rather have it ASAP as singular issues, as we need to take a look at what need to be changed or not on each of these. Also, some are already being worked on I believe like the FOV system. I'm updating your issue to reflect that.

Yes, I agree they should be made into individual issues each. I didn't want to spam a dozen new issues though especially since most of these are not a priority anytime soon. The intention is to make new issues for them once we are ready to implement them. Once an issue is made for one, it can be linked here.
